### ISSUE TITLE
Flutter 3 Warning

### ISSUE THREAD

**Issue by GodsFavoriteDeveloper** [flutter/flutter#103714]
I can't get rid of the below warning:

Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
../…/widgets/selectable.dart:459
- 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'

**Comment by newtaDev** [comment]
yes, i can confirm.
Im also getting some warning in one of my projects, 
But the application works fine

```
Launching lib/main.dart on Chrome in debug mode...
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_navigation/src/extension_navigation.dart:
357:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback((_) {
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_navigation/src/extension_navigation.dart:
468:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback((_) {
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_navigation/src/snackbar/snackbar.dart:452
:22: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    SchedulerBinding.instance!.addPostFrameCallback(
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_state_manager/src/rx_flutter/rx_disposabl
e.dart:20:22: Warning: Operand of null-aware operation '?.' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    SchedulerBinding.instance?.addPostFrameCallback((_) => onReady());
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_state_manager/src/rx_flutter/rx_notifier.
dart:130:22: Warning: Operand of null-aware operation '?.' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    SchedulerBinding.instance?.addPostFrameCallback((_) => onReady());
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_state_manager/src/simple/get_controllers.
dart:90:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.addObserver(this);
                   ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_state_manager/src/simple/get_controllers.
dart:96:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.removeObserver(this);
                   ^
../../../../../../Development/flutter/.pub-cache/git/flutterWidgets-fadd6ff9c1bee428e8b803cd4869d0b3c8545e2e/lib/src/helper/helper_w
idget/custom_expandable_page_view.dart:115:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which
excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addPostFrameCallback((_) => _notifySize());
                   ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_navigation/src/router_report.dart:53:22:
Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance!.addPostFrameCallback((_) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_screenutil-5.3.1/lib/screenutil_init.dart:23:54:
Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      data: MediaQueryData.fromWindow(WidgetsBinding.instance!.window),
                                                     ^
../../../../../../Development/flutter/.pub-cache/git/TILP-FlutterVideoplayer-4781daa15fb994f2a0e3a64948673c12eb92adec/lib/src/contro
llers/fl_video_controller.dart:39:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addPostFrameCallback((timeStamp) async {
                   ^
../../../../../../Development/flutter/.pub-cache/git/TILP-FlutterVideoplayer-4781daa15fb994f2a0e3a64948673c12eb92adec/lib/src/fl_vid
eo_player.dart:209:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {
                   ^
../../../../../../Development/flutter/.pub-cache/git/flutterWidgets-fadd6ff9c1bee428e8b803cd4869d0b3c8545e2e/lib/src/helper/helper_w
idget/custom_dropdown.dart:1218:54: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    final FocusManager focusManager = WidgetsBinding.instance!.focusManager;
                                                     ^
../../../../../../Development/flutter/.pub-cache/git/flutterWidgets-fadd6ff9c1bee428e8b803cd4869d0b3c8545e2e/lib/src/helper/helper_w
idget/custom_dropdown.dart:1225:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.removeObserver(this);
                   ^
../../../../../../Development/flutter/.pub-cache/git/flutterWidgets-fadd6ff9c1bee428e8b803cd4869d0b3c8545e2e/lib/src/helper/helper_w
idget/custom_dropdown.dart:1227:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!.focusManager
                   ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/scrollable_positioned_list-0.2.3/lib/src/scrollable_positio
ned_list.dart:439:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback((_) {
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/scrollable_positioned_list-0.2.3/lib/src/scrollable_positio
ned_list.dart:486:26: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
        SchedulerBinding.instance!.addPostFrameCallback((_) {
                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:917:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addObserver(this);
                   ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:1001:5: Warning: Operand of null-aware operation '?.' has type 'ImageCache' which excludes null.
 - 'ImageCache' is from 'package:flutter/src/painting/image_cache.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/image_cache.dart').
    imageCache?.clear();
    ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:1021:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.removeObserver(this);
                   ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:1051:5: Warning: Operand of null-aware operation '?.' has type 'ImageCache' which excludes null.
 - 'ImageCache' is from 'package:flutter/src/painting/image_cache.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/image_cache.dart').
    imageCache?.clear();
    ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:1864:42: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
                          WidgetsBinding.instance?.window.viewInsets.bottom !=
                                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:1870:48: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
                                WidgetsBinding.instance!.window.viewInsets,
                                               ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:1872:38: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
                                    .instance!.window.devicePixelRatio)
                                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:2661:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:2676:28: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
            WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                           ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:2700:26: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
          WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:2798:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:2812:26: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
          WidgetsBinding.instance
                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:3325:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/pdfviewer
.dart:3335:24: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
        WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                       ^
../../../../../../Development/flutter/.pub-cache/git/TILP-FlutterVideoplayer-4781daa15fb994f2a0e3a64948673c12eb92adec/lib/src/contro
llers/fl_getx_video_controller.dart:211:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes
null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addPostFrameCallback((timeStamp) async {
                   ^
../../../../../../Development/flutter/.pub-cache/git/TILP-FlutterVideoplayer-4781daa15fb994f2a0e3a64948673c12eb92adec/lib/src/contro
llers/fl_player_controller.dart:177:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selectable.dart:459
:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback((_) {
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/scrollable_positioned_list-0.2.3/lib/src/positioned_list.da
rt:312:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback((_) {
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/common/pd
fviewer_plugin.dart:80:5: Warning: Operand of null-aware operation '!' has type 'ImageCache' which excludes null.
 - 'ImageCache' is from 'package:flutter/src/painting/image_cache.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/image_cache.dart').
    imageCache!.clear();
    ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/common/pd
fviewer_plugin.dart:107:5: Warning: Operand of null-aware operation '!' has type 'ImageCache' which excludes null.
 - 'ImageCache' is from 'package:flutter/src/painting/image_cache.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/image_cache.dart').
    imageCache!.clear();
    ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/p
df_page_view.dart:187:21: Warning: Operand of null-aware operation '?.' has type 'PaintingBinding' which excludes null.
 - 'PaintingBinding' is from 'package:flutter/src/painting/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/binding.dart').
    PaintingBinding.instance?.imageCache?.clear();
                    ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/p
df_page_view.dart:188:21: Warning: Operand of null-aware operation '?.' has type 'PaintingBinding' which excludes null.
 - 'PaintingBinding' is from 'package:flutter/src/painting/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/binding.dart').
    PaintingBinding.instance?.imageCache?.clearLiveImages();
                    ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/p
df_page_view.dart:196:23: Warning: Operand of null-aware operation '?.' has type 'PaintingBinding' which excludes null.
 - 'PaintingBinding' is from 'package:flutter/src/painting/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/binding.dart').
      PaintingBinding.instance?.imageCache?.clear();
                      ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/p
df_page_view.dart:197:23: Warning: Operand of null-aware operation '?.' has type 'PaintingBinding' which excludes null.
 - 'PaintingBinding' is from 'package:flutter/src/painting/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/painting/binding.dart').
      PaintingBinding.instance?.imageCache?.clearLiveImages();
                      ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/p
df_scrollable.dart:380:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/overlay_m
anager.dart:138:28: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      if (SchedulerBinding.instance!.schedulerPhase ==
                           ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/overlay_m
anager.dart:140:26: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
        SchedulerBinding.instance!
                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/web_selec
tion_manager.dart:119:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/p
dfviewer_canvas.dart:793:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                     ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_pdfviewer-19.4.53-beta/lib/src/control/s
croll_head_overlay.dart:277:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addPostFrameCallback((Duration timeStamp) {
                   ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/overlay.d
art:141:26: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    if (SchedulerBinding.instance!.schedulerPhase ==
                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/overlay.d
art:143:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback(_markNeedsBuild);
                       ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/overlay.d
art:193:26: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    if (SchedulerBinding.instance!.schedulerPhase ==
                         ^
../../../../../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_math_fork-0.6.1/lib/src/widgets/selection/overlay.d
art:195:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart'
 ('../../../../../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback(_markNeedsBuild);
                       ^
Waiting for connection from debug service on Chrome...             28.7s
This app is linked to the debug service: ws://127.0.0.1:53810/dr0b6YOUrus=/ws
Debug service listening on ws://127.0.0.1:53810/dr0b6YOUrus=/ws

💪 Running with sound null safety 💪

🔥  To hot restart changes while running, press "r" or "R".
For a more detailed help message, press "h". To quit, press "q".

An Observatory debugger and profiler on Chrome is available at: http://127.0.0.1:53810/dr0b6YOUrus=
Flutter Web Bootstrap: Auto
```

> flutter doctor -v

```
[✓] Flutter (Channel stable, 3.0.0, on macOS 12.3 21E230 darwin-arm, locale en-IN)
    • Flutter version 3.0.0 at /Users/newton/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ee4e09cce0 (4 days ago), 2022-05-09 16:45:18 -0700
    • Engine revision d1b9a6938a
    • Dart version 2.17.0
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/newton/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Users/newton/Downloads/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/newton/Downloads/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.67.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.40.0

[✓] Connected device (3 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554 • android-arm64  • Android 12 (API 31) (emulator)
    • macOS (desktop)             • macos         • darwin-arm64   • macOS 12.3 21E230 darwin-arm
    • Chrome (web)                • chrome        • web-javascript • Google Chrome 101.0.4951.64

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by muezz** [comment]
I am getting the same warnings. Upon some investigation, I noticed that the files referenced in these warnings were all from third party packages. For me, they were `hooks_riverpod`, `vrouter`, `fl_chart` and `supabase_flutter`.

I ran `flutter clean` and added all of my packages again. But, by that time, only `hooks_riverpod` had one update. Once that update was done, I stopped getting warnings for that package.

In short, Flutter 3.0 changed something about how it works and these packages need to replace 
```dart
 WidgetsBinding.instance?.
// or
 WidgetsBinding.instance!.
```
to
```dart
 WidgetsBinding.instance.
```
Once each package author updates and corrects these AND you get those updated packages in your `punspec.yaml`, you will stop getting these warnings.
